packages/theme/src/cli/commands/theme/push.test.ts

```diff
@@ -97,32 +84,37 @@ describe('Push', () => {
       expect(DevelopmentThemeManager.prototype.findOrCreate).toHaveBeenCalled()
     })
 
-    test('returns live theme when live flag is provided', async () => {
+    test('creates development theme when development and unpublished flags are provided', async () => {
```

Comment:
The diff is unhappy because I re-arranged the tests.
It's probably easier to review the commits individually (They're tiny!)